feedbacks.md

```diff
@@ -0,0 +1,33 @@
+# Feedbacks Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/core/feedbacks**   
+
+As we don't have yet an use case to iterate over all the existent feedbacks the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+## Single Feedback
+**/api/core/feedbacks/<:id>**
+
+As we don't have yet an use case to get a single feedback the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error codes).
+
+## Creating a feedback
+**POST /api/core/feedbacks**
```

Comment:
NOTE to self (and other reviewers): It's good this is a `POST` request as that'll cut down on the ability to spam this endpoint... All `POST` requests require that CORS & CSRF checks will pass, so requests to this endpoint must already be coming from a "trusted" source/origin.  We will just want to makes sure the UI implementation of this feature does its best to also avoid spammers sending spam feedback.